### PR TITLE
Translated memory into Swedish.

### PR DESCRIPTION
**Pull request by brokenprogrammer**

### CODE REVIEW

File: basics/memory.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Minneshantering
+
+D är ett system programmeringsspråk och tillåter därför manuell
```
Comment (nordlow) [inline basics/memory.md]: Use `systemprogrammeringsspråk` instead.

File: basics/memory.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+hantering av minnet. Med det sagt är manuell minneshantering en källa
+till många fel, så D använder sig av en *skräpsamlare* (garbage collector) som
+standardmetod för att hantera minnetilldelningar.
```
Comment (nordlow) [inline basics/memory.md]: Use `minnestilldelningar` instead.

File: basics/memory.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+D erbjuder en pekartyp `*T` likt programmeringsspråket C:
```
Comment (nordlow) [inline basics/memory.md]: Use `likt i programmeringsspråket C`:

File: basics/memory.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+    int a;
+    int* b = &a; // b innehåller adressen till a
+    auto c = &a; // c är int* och innehåller adressen till a
+
+Ett nytt minnesblock på Heap kan allokeras med uttrycket `new`
```
Comment (nordlow) [inline basics/memory.md]: Use `på heapen`

File: basics/memory.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+som returnerar en pekare till det allokerade minnet.
+
+    int* a = new int;
+
+Så fort minnet adresserat av `a` inte länge hänvisas till av någon variabel
+i programmet, frigörs minnet av skräpsamlaren.
+
+Det finns tre säkerhetsnivåer för funktioner i D: `@system`, `@trusted` och `@safe`.
+Om inget annat anges är `@system` standard. `@safe` är en delmängd av D som förhindrar minnesfel genom att förbjuda vissa operationer. `@safe` kod kan endast anropa andra `@safe` eller `@trusted` funktioner. Samt är manuell pekararitmetik förbjuden i `@safe` kod.
```
Comment (nordlow) [inline basics/memory.md]: `Vidare är manuell pekararitmetik `  sounds better.

File: basics/memory.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+    void main() @safe {
+        int a = 5;
+        int* p = &a;
+        int* c = p + 5; // Fel
+    }
+
+`@trusted` funktioner måste verifieras manuellt av programmeraren och gör det möjligt att skapa en koppling mellan SafeD () och det underliggande osäkra, lågnivå världen.
```
Comment (nordlow) [inline basics/memory.md]: Use `den underliggande osäkra lågnivådelen av D-program`

**Review by nordlow — COMMENTED** [review]
Some nitpicks.

**Review by jacob-carlborg — APPROVED** [review]

File: basics/memory.md (hunk 6 above)
Comment (brokenprogrammer) [reply basics/memory.md]: Great comments, I'm updating right away!

**Comment by brokenprogrammer** [comment]
@nordlow Could you review the changes based on your comments? Once this is approved I will try to start working on the next section.

File: basics/memory.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
+# Minneshantering
+
+D är ett systemprogrammeringsspråk och tillåter därför manuell
+hantering av minnet. Med det sagt är manuell minneshantering en källa
+till många fel, så D använder sig av en *skräpsamlare* (garbage collector) som
+standardmetod för att hantera minnestilldelningar.
+
+D erbjuder en pekartyp `*T` likt i programmeringsspråket C:
```
Comment (Vild) [inline basics/memory.md]: `T*` not `*T`

 ``D erbjuder pekartyper `T*` likt som(?) i programmeringsspråket C:``
IMO it matches ``D provides pointer types `T*` like in C:`` better.

File: basics/memory.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+Ett nytt minnesblock på heapen kan allokeras med uttrycket `new`
+som returnerar en pekare till det allokerade minnet.
+
+    int* a = new int;
+
+Så fort minnet adresserat av `a` inte länge hänvisas till av någon variabel
+i programmet, frigörs minnet av skräpsamlaren.
+
+Det finns tre säkerhetsnivåer för funktioner i D: `@system`, `@trusted` och `@safe`.
+Om inget annat anges är `@system` standard. `@safe` är en delmängd av D som förhindrar minnesfel genom att förbjuda vissa operationer. `@safe` kod kan endast anropa andra `@safe` eller `@trusted` funktioner. Vidare är manuell pekararitmetik förbjuden i `@safe` kod.
```
Comment (Vild) [inline basics/memory.md]: `Vidare är manuell pekararitmetik förbjuden i `@safe` kod:` should end with a `:` to match the english document.

Nitpick: Maybe also not write all of these sentences on the same line. But this doesn't really matter.

File: basics/memory.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+
+    void main() @safe {
+        int a = 5;
+        int* p = &a;
+        int* c = p + 5; // Fel
+    }
+
+`@trusted` funktioner måste verifieras manuellt av programmeraren och gör det möjligt att skapa en koppling mellan SafeD () och den underliggande osäkra lågnivådelen av D-program.
```
Comment (Vild) [inline basics/memory.md]: Remove `()`

File: basics/memory.md [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+* [SafeD](https://dlang.org/safed.html)
+
+## {SourceCode}
+
+```d
+import std.stdio : writeln;
+
+void safeFun() @safe
+{
+    writeln("Hello World");
```
Comment (Vild) [inline basics/memory.md]: Nitpick: Change to `Hej Världen!` to match welcome-to-d.md

**Review by Vild — CHANGES_REQUESTED** [review]

File: basics/memory.md (hunk 10 above)
Comment (brokenprogrammer) [reply basics/memory.md]: Great! I'm updating right away.

File: basics/memory.md [hunk 11]
```diff
@@ -5,7 +5,7 @@ hantering av minnet. Med det sagt är manuell minneshantering en källa
 till många fel, så D använder sig av en *skräpsamlare* (garbage collector) som
 standardmetod för att hantera minnestilldelningar.
 
-D erbjuder en pekartyp `*T` likt i programmeringsspråket C:
+D erbjuder en pekartyper `T*` likt som i programmeringsspråket C:
```
Comment (Vild) [inline basics/memory.md]: Remove `en`

Comment (brokenprogrammer) [reply basics/memory.md]: Ah of course, my bad! Shouldn't have commited so late at night.

**Review by Vild — APPROVED** [review]

**Pull request merged**